src/go/k8s/pkg/resources/certmanager/keystore_password.go

```diff
@@ -52,14 +52,14 @@ func NewKeystoreSecretResource(
 }
 
 // Ensure will manage pkcs#12 and jks password for KafkaAPI certificate
-func (r *KeystoreSecretResource) Ensure(ctx context.Context) error {
+func (r *KeystoreSecretResource) Ensure(ctx context.Context) (bool, error) {
 	obj, err := r.obj()
 	if err != nil {
-		return fmt.Errorf("unable to construct object: %w", err)
+		return false, fmt.Errorf("unable to construct object: %w", err)
 	}
 
 	_, err = resources.CreateIfNotExists(ctx, r, obj, r.logger)
-	return err
+	return err == nil, err
```

Comment:
I was checking `CreateIfNotExists` implementation, and looks like it can return `false, nil`.
In that case I think we should also return `false, nil` in here.
WDYT?